app/grandchallenge/components/schemas.py

```diff
@@ -491,6 +491,67 @@
             "required": ["version", "type", "ellipses"],
             "additionalProperties": False,
         },
+        "3-point-angle-object": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"const": "Three-point angle"},
+                "angle": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": {"type": "number"},
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 3,
+                    "maxItems": 3,
+                },
+                "probability": {"type": "number", "minimum": 0, "maximum": 1},
+            },
+            "required": ["angle"],
+            "additionalProperties": False,
+        },
+        "3ANG": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"const": "Three-point angle"},
+                "angle": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": {"type": "number"},
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 3,
+                    "maxItems": 3,
+                },
+                "version": {"$ref": "#/definitions/version-object"},
+                "probability": {"type": "number", "minimum": 0, "maximum": 1},
+            },
+            "required": ["angle"],
```

Comment:
Shouldn't `"version"` also be required?

Reply:
Reviewing this myself yields that you are right! For the "3ANG", this should be required. The "xxx-object" types do not have those... fixing!

Reply:
> The "xxx-object" types do not have those... 

I think that is correct, they're only used in the Multi-x-object, the versioning on those is done at the top level.